administrative-sdk/student.js

```diff
@@ -123,63 +80,41 @@ class Student {
    * @param {Sdk~getCallback} [cb] The callback that handles the response.
    * @param {Sdk~getErrorCallback} [ecb] The callback that handles the error response.
    */
-  static getStudent(connection, organisationId, studentId, cb, ecb) {
-    var _cb = function(data) {
-      var student = new Student(organisationId, data.id, data.firstName,
-        data.lastName, data.gender, data.birthYear);
-      student.created = new Date(data.created);
-      student.updated = new Date(data.updated);
-      if (cb) {
-        cb(student);
-      }
-    };
-
+  static getStudent(connection, organisationId, studentId) {
     var url = connection.settings.apiUrl + '/organisations/' +
       organisationId + '/students/' + studentId;
-    connection._secureAjaxGet(url, _cb, ecb);
+    return connection._secureAjaxGet(url)
+      .then(data => {
+        var student = new Student(organisationId, data.id, data.firstName,
+          data.lastName, data.gender, data.birthYear);
+        student.created = new Date(data.created);
+        student.updated = new Date(data.updated);
+        return student;
+      });
   }
 
   /**
-   * Callback used by listStudents.
-   *
-   * @callback Sdk~listCallback
-   * @param {its.Student[]} student Retrieved student domain model instances.
-   */
-  studentListCallback(student) {}
-
-  /**
-   * Error callback used by listStudents.
-   *
-   * @callback Sdk~listErrorCallback
-   * @param {object[]} errors Array of errors.
-   */
-  studentListErrorCallback(errors) {}
-
-  /**
    * List all students in the organisation.
    *
    * @param {string} organisationId Specify an organisation identifier.
    * @param {Sdk~listCallback} cb The callback that handles the response.
    * @param {Sdk~listErrorCallback} [ecb] The callback that handles the error response.
```

Comment:
Missing documentation update. Callbacks are no longer taken and a promise is returned.

I think you're getting the idea now... `git grep` is your friend.